scripts/tokenomics-emulation/emulations/transfer-native-token.emulation.ts

```diff
@@ -11,21 +11,35 @@ class NativeTokensTransferEmulation implements IEmulation {
   ) {}
 
   public async run(): Promise<void> {
-    for (let i = 1; i <= this.config.amount; i++) {
-      console.log(`Running ${i} native token transfer...\n`);
-      const destination = await this.account.generateSrAccount();
-      const transferAmount = _.random(
-        this.config.tokens_range[0],
-        this.config.tokens_range[1]
-      );
-      const sender = this.account.rootAccount;
-      const transfer = await this.network.transfer(
-        sender,
-        destination.ss58Address,
-        transferAmount.toString()
+    const total = this.config.amount;
```

Comment:
We need to reuse this logic in different emulations. Move it to a separate class `Batcher`.

Reply:
Added a new class Batcher